types/react-relay/test/react-relay-tests.tsx

```diff
@@ -717,7 +717,7 @@ requestSubscription(
 ReactRelayContext.Consumer.prototype;
 ReactRelayContext.Provider.prototype;
 
-const MyRelayContextProvider: React.FunctionComponent = ({children}) => {
+const MyRelayContextProvider: React.FunctionComponent<{ children?: React.ReactNode }> = ({children}) => {
```

Comment:
This is already the case for FunctionComponents? 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8287af144c52c1f85de638aff2142f4219698643/types/react/index.d.ts#L550


https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8287af144c52c1f85de638aff2142f4219698643/types/react/index.d.ts#L829

Or is the test just explicitly making sure it still works? 😅 just trying to understand

Reply:
> This is already the case for FunctionComponents?

This is going to be removed from `FunctionComponent`. I'm filing it now so that we don't end up with a PR that pings hundreds of people since it touches multiple packages.